app/scripts/modules/core/src/help/help.contents.ts

```diff
@@ -265,6 +265,7 @@ module(HELP_CONTENTS, [])
         <ul><li><b>Oldest</b> will terminate the oldest instances first</li>
         <li><b>Newest</b> will terminate those most recently launched.</li></ul></p>`,
     'strategy.rollingRedBlack.targetPercentages': '<p>Rolling red black will slowly scale up the new server group. It will resize the new server group by each percentage defined.</p>',
+    'strategy.rollingRedBlack.rollback': '<p>Disable the new server group and ensure that the previous server group is sized correctly.</p>',
```

Comment:
do you anticipate adding this checkbox to other strategies, and if so do you think they'll need different help messages?

Reply:
What does "correctly" mean as far as the size of the previous server group? Is it set to its original size when the operation started?

Reply:
@christopherthielen Yes, each strategy will have a strategy-specific help message. For example, automatically rolling back a highlander may involve actually destroying the newly launched server group. 

@anotherchrisberry Yes, effectively it's min capacity will be unpinned and restored to it's original value.